### PR TITLE
Removed old version

### PR DESCRIPTION
**Pull request by guettli**
The getting-started-guide should be nice and clean.

Those people using these dated software can find help elsewhere.

### CODE REVIEW

**Comment by stgraber** [comment]
Hey, can you update your commit message to include the required `Signed-off-by` line?
This can usually be done with `git commit --amend -s && git push --force`

File: content/lxd/getting-started-cli.md [hunk 1]
```diff
@@ -409,4 +395,4 @@ You find more information on the following pages:
  
  <!-- footnotes -->
  
- [^1]: [Running virtual machines with lxd](https://discuss.linuxcontainers.org/t/running-virtual-machines-with-lxd-4-0/7519), including a short howto for a Microsoft Windows VM.
\ No newline at end of file
+ [^1]: [Running virtual machines with lxd](https://discuss.linuxcontainers.org/t/running-virtual-machines-with-lxd-4-0/7519), including a short howto for a Microsoft Windows VM.
```
Comment (toby63) [inline content/lxd/getting-started-cli.md:398]: Why is this replaced?

**Comment by toby63** [comment]
@guettli I included your changes in my PR #472. I hope thats ok, I included you as co-author, I hope you are ok with that as well.
If not just comment on #472.

**Comment by stgraber** [comment]
Closing as included in other PR

**Pull request closed without merging**